concepts/Common/Editors Concepts/How to Choose a Drop-Down Editor.md

```diff
@@ -83,7 +83,7 @@ The following table compares the drop-down editors according to features:
     </tr>
 </table>
 
-DropDownBox allows you to put any content in its drop-down field. This feature makes it more versatile but it is also more complex to implement. For instance, you can embed another DevExtreme component and use it to display data. Refer to the DropDownBox [help topics](/concepts/05%20UI%20Components/DropDownBox/10%20Overview.md '/Documentation/Guide/UI_Components/DropDownBox/Overview/') and [demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) for further information.
+DropDownBox allows you to put any content in its drop-down field. This feature makes it more versatile but it is also more complex to implement. For instance, you can embed another DevExtreme component and use it to display data. Refer to the DropDownBox [help topics](/Documentation/Guide/UI_Components/DropDownBox/Getting_Started_with_DropDownBox/) and [demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) for further information.
```

Comment:
> This feature makes **it** more versatile but **it** is also more complex to implement.

Too many pronouns in a row. We are already losing the thing they refer to. Does the drop-down field become more versatile or?

It's probably better to rephrase the entire paragraph.

Reply:
Okay, I will rewrite it

Reply:
OK